### PR TITLE
Add tex-unpack-params.html

### PR DESCRIPTION
**Pull request by zhenyao**
This tests all ES3 unpack parameters.

Note that this is not an fully implemented test yet. I plan to add pixel
verification in a followup CL.

### CODE REVIEW

File: sdk/tests/conformance2/textures/misc/tex-unpack-params.html [hunk 1]
```diff
@@ -0,0 +1,409 @@
...
+function runTests() {
+  var gl = wtu.create3DContext("example");
+  if (!gl) {
+    testFailed("Fail to get a WebGL context");
+    return;
+  }
+
+  // For 2D cases, depth, zoffset, imageHeight, skipImages are ignored.
+  var testCases = [
+    { width: 5, height: 7, depth: 3, xoffset: 2, yoffset: 3, zoffset: 1,
```
Comment (toji) [inline sdk/tests/conformance2/textures/misc/tex-unpack-params.html]: This is getting a bit pedantic, but it would be nice to see some tests with different width/height/depth/offsets, just to make sure we don't happen to be hitting some really weird edge case. Specifically it would be good to do a few tests where the textures are powers of 2. That should be the "simple" case, but it's worth testing.

Also, if these are going to be the same for a large number of cases in may be better to patch them in down below rather than repeat them over and over. For example:

```
for (var ii = 0; ii < testCases.length; ++ii) {
    var testCase = testCases[ii];
    testCase.width = 5;
    testCase.height = 7;
    // Etc.
    runTestIteration2D(gl, testCase, useUnpackBuffer);
    runTestIteration3D(gl, testCase, useUnpackBuffer);

    testCase.width = 4;
    testCase.height = 8;
    // Etc.
    runTestIteration2D(gl, testCase, useUnpackBuffer);
    runTestIteration3D(gl, testCase, useUnpackBuffer);
 }
```

**Comment by toji** [comment]
LGTM with one suggestion.

File: sdk/tests/conformance2/textures/misc/tex-unpack-params.html (hunk 1 above)
Comment (zhenyao) [inline sdk/tests/conformance2/textures/misc/tex-unpack-params.html]: Good suggestion.  I'll add a few more cases.

**Comment by kenrussell** [comment]
Very nice test @zhenyao and good suggestion @toji . No further comments aside from those. LGTM after the fact.
